src/overview.md

```diff
@@ -89,85 +89,88 @@ class Point {
-  threads but don't share memory, communicating only via messages
+  threads but don't share memory, communicating only through messages
   (`dart:isolate`)
 - HTML elements and other resources for web-based applications that need to
-  interact with the browser and the DOM (Document Object Model) (`dart:html`)
+  interact with the browser and the Document Object Model (DOM) (`dart:html`)
 
 Our [library tour](https://dart.dev/guides/libraries/library-tour) demonstrates
-some of the classes provided in Dart through a series of worked examples.
+some of the classes provided in Dart through a series of working examples.
 
 Beyond the core libraries, many APIs are provided through a comprehensive set of
-packages. Dart itself offers many useful supplementary packages including
-[characters](https://pub.dev/packages/characters),
-[intl](https://pub.dev/packages/intl), [http](https://pub.dev/packages/http),
-[crypto](https://pub.dev/packages/crypto) and
-[markdown](https://pub.dev/packages/markdown). In addition, there are many
-thousands of packages from third-party publishers and the broader community,
-including packages for [XML](https://pub.dev/packages/xml), [Windows
-integration](https://pub.dev/packages/win32),
-[SQLite](https://pub.dev/packages/sqflite_common) and [compression]
-(https://pub.dev/packages/archive).
-
-## Dart: The Platform
+packages. Dart also offers many useful supplementary packages, including the following:
+- [characters](https://pub.dev/packages/characters)
+- [intl](https://pub.dev/packages/intl) 
+- [http](https://pub.dev/packages/http)
+- [crypto](https://pub.dev/packages/crypto)
+- [markdown](https://pub.dev/packages/markdown)
+
+In addition, there are thousands of packages from third-party publishers and the broader community,
+including packages for the following:
+- [XML](https://pub.dev/packages/xml) 
+- [Windows integration](https://pub.dev/packages/win32)
+- [SQLite](https://pub.dev/packages/sqflite_common)
+- [compression](https://pub.dev/packages/archive)
+
+## Dart: The platform
 
 Dart's compiler technology lets you run code in different ways:
 
 - **Native compilation**: For apps targeting mobile and desktop devices, Dart
-  includes both a Dart VM with JIT (just-in-time) compilation and an AOT
-  (ahead-of-time) compiler for producing machine code.
+  includes both a Dart VM with just-in-time(JIT) compilation and an 
+  ahead-of-time (AOT) compiler for producing machine code.
 
-- **Web compilation**: For apps targeting the web, Dart also includes both a
-  development time compiler (dartdevc) and a production time compiler (dart2js),
-  both of which translate Dart into JavaScript that is supported on all
+- **Web compilation**: For apps targeting the web, Dart includes both a
+  development time compiler (dartdevc) and a production time compiler (dart2js).
+  Both compilers translate Dart into JavaScript, which is supported on all
```

Comment:
Should this be "that's supported on [all major browsers](/faq#q-what-browsers-do-you-support-as-javascript-compilation-targets)?

Because we technically don't support all major browsers, according to https://dart.dev/faq#q-what-browsers-do-you-support-as-javascript-compilation-targets.